scripts/build-desktop.sh

```diff
@@ -13,6 +13,16 @@ else
   ENV_FILE=".env"
 fi
 
+if [[ -n $GCP_GEOLOCATION_API_KEY ]]; then
+  if grep -q "^GCP_GEOLOCATION_API_KEY=" "$ENV_FILE"; then
+      # Replace the value for the existing key
+      sed -i "s|^GCP_GEOLOCATION_API_KEY=.*$|GCP_GEOLOCATION_API_KEY=$GCP_GEOLOCATION_API_KEY|g" "$ENV_FILE"
```

Comment:
good thought, but I think that would probably break things because react-native-config and react-web-config don't strip quotes from values. You can see that the existing .env files don't have any quoted values: https://github.com/Expensify/App/blob/cd8cb9bf2b1323476f4fe9af89199716f095ca84/.env.staging

Reply:
Surprising but ok good to know 👍